docs/release/trg-1/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Architecture Documentation
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 30th of March 2024
+:::
+
+| Status   | Created     | Post-History                                          |
+|----------|-------------|-------------------------------------------------------|
+| Active   | 22-Apr-2024 |                                                       |
+| Draft    | 26-Mar-2024 | Transferred from Release Acceptance Criteria into TRG |
+
+## Why
+
+Proper Architecture Documentation is necessary to support architects and operation teams for their decisions making processes and operative tasks. It therefore provides information like context, integration interfaces and intended usage of the component. We follow the good practice to document the Architecture of components.
+
+## Description
+
+Eclipse Tractus-X is using the [arc42](https://arc42.org/) documentation template. If you are not yet familiar with it, see [arc42.org](https://arc42.org/) for an introduction and guidance.
+
+In order to pragmatically start or update your Architecture documentation, consult [arc42](https://arc42.org/). Architecture documentation **must** be maintained in a directory named `/docs/architecture` in the leading product repository. Within the Architecture folder the structure of the arc42 template **shall** be applied. Follow the guidance to understand, WHAT and HOW to document.
+
+- The Architecture Documentation **must** be stored in the folder: `/docs/architecture`
+  - It is up-to-date for every release
+  - English is a **must**
+- The `/docs/README.md` describes shortly what documentation is available
+
+## Best Practice
+
+- Provide each chapter of the arc42 documentation as single `.md` file named after the chapter
+- Favor diagrams as mermaid block in `.md` files directly over image files, where possible
+- Have a look to peer repositories within Tractus-X for inspiration
```

Comment:
Dokchess is a great arc42 example documentation made by Stefan Zörner, one of the initial creators of arc42. I would vote for mentioning this example here for inspiration: https://www.dokchess.de/en/

Reply:
@mhellmeier i can't determine if it makes sense or not but i would like to ask you to create a new PR after to add it as this is not my PR (the original text at least) and i need to get this cleaned up / merged before i have no longer any resources

Reply:
I will solve it with an inline suggestion, no problem!